src/openforms/formio/components/np_family_members/tests/test_family_members.py

```diff
@@ -150,6 +150,31 @@ def test_get_partners_stuf_bg(self, mock_stufbg_config_get_solo):
             self.assertEqual(("123123123", "Belly van Doe"), partners_choices[0])
             self.assertEqual(("456456456", "Bully van Doe"), partners_choices[1])
 
+    @tag("gh-3864")
+    @patch("stuf.stuf_bg.client.StufBGConfig.get_solo")
+    def test_get_single_partner_stuf_bg(self, mock_stufbg_config_get_solo):
+        stuf_bg_service = StufServiceFactory.build()
+        mock_stufbg_config_get_solo.return_value = StufBGConfig(service=stuf_bg_service)
+        soap_response_template = (
+            TEST_FILES / "stuf_bg_one_family_member.xml"
+        ).read_text()
+        response_content = render_from_string(soap_response_template, {}).encode(
+            "utf-8"
+        )
+
+        with requests_mock.Mocker() as m:
+            m.post(
+                stuf_bg_service.get_endpoint(type=EndpointType.vrije_berichten),
+                content=response_content,
+            )
+
+            partners_choices = get_np_family_members_stuf_bg(
+                bsn="111222333", include_children=False, include_partners=True
```

Comment:
`include_children=False` should result in no children being returned in the response data, no? Since we don't request it...